src/app/organizations/vault/add-edit.component.ts

```diff
@@ -55,14 +55,14 @@ export class AddEditComponent extends BaseAddEditComponent {
     }
 
     protected loadCollections() {
-        if (!this.organization.canManageAllCollections) {
+        if (!this.organization.canEditAnyCollection) {
```

Comment:
I know this wasn't really changed in your code, but would it be possible to just check if `this.collections` is set and if so just return it? Since we only set it when `canEditAnyCollection == true`.

Reply:
This is a confusing bit of logic, but we can't do that without some pretty big refactors.

What's going on is this:
1. `collections` defaults to an empty list
2. the organization vault component has extra collections context and so takes the opportunity to set `collections` to all collections if it can.
3. the organization add-edit component sets its own collections otherwise.